ontology/yaml/resources/METERS/entity_types/WM.yaml

```diff
@@ -36,3 +36,15 @@ WM_WF:
   - supply_water_volume_accumulator
   - water_linearvelocity_sensor
   - water_volume_accumulator
+
+WM_WFM:
```

Comment:
Please see above and use the existing `WM_WF` type.

Reply:
yes, i guess it's almost the same. Thanks.